[2025-05-12T07:59:04.707+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:59:04.195219+00:00 [queued]>
[2025-05-12T07:59:04.711+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:59:04.195219+00:00 [queued]>
[2025-05-12T07:59:04.711+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-05-12T07:59:04.716+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): extract_breweries> on 2025-05-12 07:59:04.195219+00:00
[2025-05-12T07:59:04.719+0000] {standard_task_runner.py:57} INFO - Started process 1961 to run task
[2025-05-12T07:59:04.721+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_brewery', 'extract_breweries', 'manual__2025-05-12T07:59:04.195219+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/brewery_pipeline.py', '--cfg-path', '/tmp/tmpktzf0lti']
[2025-05-12T07:59:04.723+0000] {standard_task_runner.py:85} INFO - Job 13: Subtask extract_breweries
[2025-05-12T07:59:04.753+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_brewery.extract_breweries manual__2025-05-12T07:59:04.195219+00:00 [running]> on host 0d8312ffa556
[2025-05-12T07:59:04.791+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_brewery' AIRFLOW_CTX_TASK_ID='extract_breweries' AIRFLOW_CTX_EXECUTION_DATE='2025-05-12T07:59:04.195219+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-12T07:59:04.195219+00:00'
[2025-05-12T07:59:04.792+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-12T07:59:04.793+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python3 /opt/***/scripts/extract_breweries.py']
[2025-05-12T07:59:04.797+0000] {subprocess.py:86} INFO - Output:
[2025-05-12T07:59:05.092+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2025-05-12T07:59:05.093+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/extract_breweries.py", line 27, in <module>
[2025-05-12T07:59:05.093+0000] {subprocess.py:93} INFO -     run()
[2025-05-12T07:59:05.094+0000] {subprocess.py:93} INFO -   File "/opt/***/scripts/extract_breweries.py", line 19, in run
[2025-05-12T07:59:05.095+0000] {subprocess.py:93} INFO -     with open(filename, "w") as f:
[2025-05-12T07:59:05.095+0000] {subprocess.py:93} INFO - FileNotFoundError: [Errno 2] No such file or directory: 'opt/***/data_lake/bronze/breweries_12052025_075905.json'
[2025-05-12T07:59:05.109+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-05-12T07:59:05.118+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/opt/airflow/.local/lib/python3.9/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-05-12T07:59:05.120+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=etl_brewery, task_id=extract_breweries, execution_date=20250512T075904, start_date=20250512T075904, end_date=20250512T075905
[2025-05-12T07:59:05.127+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 13 for task extract_breweries (Bash command failed. The command returned a non-zero exit code 1.; 1961)
[2025-05-12T07:59:05.162+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-05-12T07:59:05.186+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
